db/go/20201008153033_update_jupyter_lab_template.go

```diff
@@ -0,0 +1,170 @@
+package migration
+
+import (
+	"database/sql"
+	uid2 "github.com/onepanelio/core/pkg/util/uid"
+	"github.com/pressly/goose"
+)
+
+const jupyterWorkspaceTemplate4 = `
+# Docker containers that are part of the Workspace
+containers:
+- name: jupyterlab-tensorflow
+  image: onepanel/jupyterlab:1.0.1
+  command: ["/bin/bash", "-c", "pip install onepanel-sdk && start.sh jupyter lab --LabApp.token='' --LabApp.allow_remote_access=True --LabApp.allow_origin=\"*\" --LabApp.disable_check_xsrf=True --LabApp.trust_xheaders=True --LabApp.base_url=/ --LabApp.tornado_settings='{\"headers\":{\"Content-Security-Policy\":\"frame-ancestors * \'self\'\"}}' --notebook-dir='/data' --allow-root"]
+  env:
+    - name: tornado
+      value: "'{'headers':{'Content-Security-Policy':\"frame-ancestors\ *\ \'self'\"}}'"
+  args:
+  ports:
```

Comment:
@aleksandrmelnikov Might be pasting issues, but I think ports needs to be indented below args?

Reply:
My original intent was to have args be blank. I realize now it can be removed completely, so I removed it.